homeassistant/components/sensor/simulated.py

```diff
@@ -79,11 +87,14 @@ def __init__(self, name, unit, amp, mean, period, phase, fwhm, seed):
         self._seed = seed
         self._random = Random(seed)  # A local seeded Random
         self._start_time = dt_util.utcnow()
+        self._relative_to_epoch = relative_to_epoch
         self._state = None
 
     def time_delta(self):
         """Return the time delta."""
         dt0 = self._start_time
+        if self._relative_to_epoch:
```

Comment:
This is the wrong implementation. We only need to calculate `self._start_time` once, and that's in the constructor.

Reply:
Thanks, I should've fixed this now.